scripts/ci_build_and_test_in_container.sh

```diff
@@ -259,6 +263,12 @@ if [[ "${RUN_UNIT_TESTS}" = true ]]; then
   fi
 fi
 
+if [[ "${RUN_PYGEOSX_TESTS}" = true ]]; then
+  or_die ninja pygeosx
+  or_die ninja geosx_python_tools
+  or_die ninja pygeosx_unit_tests
+fi
```

Comment:
I think this should go aside the `if [[ "${BUILD_EXE_ONLY}" = true ]]; then` block, before the `sccache` block for example.